docs/source/usage/plugins/phaseSpace.rst

```diff
@@ -37,47 +37,61 @@ Option                                 Usage
 Output
 ^^^^^^
 
-Values are given as *charge density* per phase space bin.
-In order to scale to a simpler *charge of particles* per :math:`\mathrm{d}r_i` and :math:`\mathrm{d}p_i` -bin multiply by the cell volume:
+The histograms are stored in ``.hdf5`` files in the ``simOutput/...`` directory.
 
-.. code:: python
+...
 
-   ps = ... # get data set from h5py
+Values are given as *charge density* per phase space bin.
+In order to scale to a simpler *charge of particles* per :math:`\mathrm{d}r_i` and :math:`\mathrm{d}p_i` -bin multiply by the cell volume ``dV``.
```

Comment:
OPTIONAL:
space between p_i and -bin could be removed

Reply:
I had this since the bin is spanned by drdp and I did not want to raise the impresion its a dp-bin

Reply:
Okay